partitions.go

```diff
@@ -231,13 +231,17 @@ func checkBootEnvAndRootPartitionMatch(bootPartNum string, rootPart string) bool
 	return strings.HasSuffix(rootPart, bootPartNum)
 }
 
-func resolveLink(path string) string {
+func resolveLink(unresolvedPath string) string {
 	// If the supplied path is not a link the original path is returned
-	resolvedPath, err := filepath.EvalSymlinks(path)
+	resolvedPath, err := filepath.EvalSymlinks(unresolvedPath)
 	// This would only happen if supplied a link that goes nowhere or creates a loop
 	if err != nil {
-		log.Warnf("Could not resolve path link: %s Attempting to continue", path)
-		return path
+		log.Warnf("Could not resolve path link: %s Attempting to continue", unresolvedPath)
+		return unresolvedPath
 	}
-	return (resolvedPath)
+	// Only resolve /dev/disk/by-partuuid/
```

Comment:
I would probably extend this comment with a why, for future you and @kacf  :). But up to you, the trace ability back to the original discussion is there

Reply:
+1